setup.py

```diff
@@ -149,6 +149,7 @@ def run_tests(self):
         versioneer.get_cmdclass(),
         {"test": NoseTestCommand}
     ]], [])),
+    conda_features=[],
```

Comment:
`conda-build` 2.1.1 has improved the situation on this front. So we shouldn't need this workaround any more.